float64[] linear_action_list
float64[] angular_action_list
bool[] enable_list
---
bool success